gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/pod/PodDistributionPluginTest.groovy

```diff
@@ -0,0 +1,235 @@
+package com.palantir.gradle.dist.pod
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.palantir.gradle.dist.GradleTestSpec
+import org.gradle.testkit.runner.BuildResult
+
+class PodDistributionPluginTest extends GradleTestSpec {
+
+    def 'manifest file contains expected fields'() {
+        given:
+        createUntarBuildFile(buildFile)
+        buildFile << '''
+            distribution {
+                serviceName 'pod-name'
+            }
+        '''.stripIndent()
+
+        when:
+        runSuccessfully(':configTar', ':untar')
+
+        then:
+        String manifest = file('dist/pod-name-0.0.1/deployment/manifest.yml', projectDir).text
+        manifest.contains('"manifest-version": "1.0"')
+        manifest.contains('"product-group": "service-group"')
+        manifest.contains('"product-name": "pod-name"')
+        manifest.contains('"product-version": "0.0.1"')
+        manifest.contains('"product-type": "pod.v1"')
+    }
+
+    def 'pod file contains expected fields'() {
+        given:
+        createUntarBuildFile(buildFile)
+        buildFile << '''
+            distribution {
+                serviceName 'pod-name'
+                
+                service "bar-service", {
+                  productGroup = "com.palantir.foo"
+                  productName = "bar"
+                  productVersion = "1.0.0"
+                  volumeMap = ["bar-volume": "random-volume"]
+                }
+                service "baz-service", {
+                  productGroup = "com.palantir.foo"
+                  productName = "baz"
+                  productVersion = "1.0.0"
+                  volumeMap = ["baz-volume": "random-volume"]
+                }
+                
+                volume "random-volume", {
+                  desiredSize = "10 GB"
```

Comment:
Can we regex validate this string a little bit? I'm concerned that people will write "10gb", "10 GB", "10 Gib" or something silly...